README.md

```diff
@@ -152,14 +152,47 @@ public class Startup
 {
   public void ConfigureServices(IServiceCollection services)
   {
-      services.AddFeatureManagement();
+      services.AddFeatureManagement()
+              .AddFeatureFilter<MyCriteriaFilter>();
   }
 }
 ```
 
-This tells the feature manager to use the "FeatureManagement" section from the configuration for feature flag settings.
+This tells the feature manager to use the "FeatureManagement" section from the configuration for feature flag settings. It also registers a customized feature filter `MyCriteriaFilter`. Please refer to the `Implementing a Feature Filter` section below for more details.

```

Comment:
Why bring up filters here? It's a bit early.

Reply:
I mentioned that "By default, the feature manager and feature filters should be registered as singleton services by calling AddFeatureManagement." So I bring up feature filter here.

Reply:
The concept of feature filter has been introduced in the `Feature Filters` section. Would it be fine to bring up filters here?